action-server/tests/actions/validate_action_test_helper.py

```diff
@@ -41,19 +42,59 @@ def create_tracker(
             last_action,
         )
 
-    def check_slot_value_accepted(self, slot_name: str, value: Any) -> None:
-        self.check_slot_value_stored(slot_name, value, value)
+    def check_slot_value_accepted(
+        self,
+        slot_name: str,
+        value: Any,
+        extra_events: List[EventType] = None,
+        previous_slots: dict = None,
+        templates: List[str] = None,
```

Comment:
Using None everywhere for this (for those who don't know): https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/#:~:text=In%20Python%2C%20when%20passing%20a,or%20even%20a%20class%20instance. We shouldn't change these arguments, since they are only used for comparison, but better be safe than sorry.